source/extensions/filters/common/ratelimit_config/ratelimit_config.cc

```diff
@@ -0,0 +1,141 @@
+#include "source/extensions/filters/common/ratelimit_config/ratelimit_config.h"
+
+#include "source/common/config/utility.h"
+#include "source/common/http/matching/data_impl.h"
+#include "source/common/matcher/matcher.h"
+
+namespace Envoy {
+namespace Extensions {
+namespace Filters {
+namespace Common {
+namespace RateLimit {
+
+RateLimitPolicy::RateLimitPolicy(const ProtoRateLimit& config,
+                                 Server::Configuration::CommonFactoryContext& context,
+                                 absl::Status& creation_status, bool no_limit) {
+  if (config.has_stage() || !config.disable_key().empty()) {
+    ENVOY_LOG(warn, "'stage' field and 'disable_key' field are not supported");
+  }
+
+  if (config.has_limit()) {
+    if (no_limit) {
+      ENVOY_LOG(warn, "'limit' field is only supported in filter that calls remote rate "
+                      "limit service.");
+    }
+  }
```

Comment:
Should these return errors that lead to config rejection?

Reply:
Configuration rejection actually bring some risks in production env.

So, basically I inclined to flush warning log because they won't effect the feature.

Reply:
People generally don't notice warnings. For this case I would make it a failure. People should be checking for config failures.

Reply:
Got it.